.gitignore

```diff
@@ -34,3 +34,6 @@ __pycache__
 localfileorder.json
 
 logback.log
+
+# Ignore folder created when substituting receiver fields in MSH Header
+substituted_receivers
```

Comment:
Looks like this folder was renamed to `substituted_header`?

Reply:
Can the temporary file be deleted after calling the hurl command instead?

Reply:
I implemented your suggestion